client/src/components/EventCard.vue

```diff
@@ -1,21 +1,17 @@
 <template>
   <v-card class="mx-3 elevation-0" :class="backgroundColour">
-    <v-card-title>
-      <span class="font-weight-bold">{{ event.name }}</span>
-      <v-spacer />
-      <v-icon
-        v-if="user?.teamAdmin && !event.isPublic"
-        icon="mdi-pencil"
-        @click="openModal"
```

Comment:
shouldnt we keep the v-if? We dont want any user editing public events and we dont want users that arent team admins to edit private events.

Reply:
Oops, I had put the v-if in a v-col containing the icon, but when I got rid of the v-col I must have forgotten to put it back. Will fix.